.ddev/.disabled-services/docker-compose.chromedriver.yaml

```diff
@@ -0,0 +1,30 @@
+---
+# Docker-ChromeDriver configuration for DDEV-Local.
```

Comment:
Off topic, but seeing this you may be interested in https://github.com/lullabot/drainpipe and https://github.com/lullabot/drainpipe-dev.

Reply:
For this project I'd be interested in the composer-lock-diff integration.